src/main/steps/common/form/fields.njk

```diff
@@ -69,6 +72,22 @@
         items: formItems(field.values, formState[fieldName]),
         errorMessage: getError(fieldName)
       }) }}
+    {% case 'textarea' %}
```

Comment:
I wonder if we should split out each case into it's own file

Reply:
👍 Good idea, I've moved all of these out into seperate files